src/cdn_definitions/_impl/__init__.py

```diff
@@ -1,26 +1,103 @@
 import json
 import os
+import warnings
+
+import requests
 import yaml
+from urllib3.util.retry import Retry
+
 
+def to_dict(data, ext):
+    """
+    Converts a string or a requests.Response object to a dict.
```

Comment:
Not a big deal, but the function name and doc string aren't actually true since JSON or YAML files can contain kinds of data other than a dict.  e.g. a file containing literally `1` is valid JSON and valid YAML and would cause this function to return an integer.  I'd have named it something like `parsed` since it's parsing the input as JSON or YAML but not converting it to any particular type.

Reply:
Ah, that's true. Changed the function name and doc string.